hybridse/using-zetasql.md

```diff
@@ -0,0 +1,294 @@
+| ResolvedFunctionCallBase                                     |                  |            |
+| ASTCastExpression                                            | CastExprNode     |            |
+| ASTCaseValueExpression                                       | CaseWhenExprNode |            |
+| ASTFunctionCall                                              | CallExprNode     |            |
+| ASTBinaryExpression <br/>ASTAndExpr <br/>                    | BinaryExpr       |            |
+| ASTUnaryExpression                                           | UnaryExpr        |            |
+| ASTIdentifier                                                | ExprIdNode       |            |
+| ASTPathExpression                                            | ColumnRefNode    |            |
+| ASTBetweenExpression                                         | BetweenExpr      |            |
+| ASTOrderBy                                                   | OrderByNode      |            |
+| ASTTablePathExpression                                       | TableRefNode     |            |
+|                                                              |                  |            |
+|                                                              |                  |            |
+
+ #### Convert `ParserOutput` to LogicalPlan  [WIP]
+
+| ParserOutput / ASTNode                                       | SQLNode         | LogicalPlan                                                  |
+| ------------------------------------------------------------ | --------------- | ------------------------------------------------------------ |
+| ASTSelect  <br/>ASTSelectList <br/>ASTSelectColumn <br/>ASTFromClause <br/>ASTWhereClause | SelectQueryNode | `bool Planner::CreateSelectQueryPlan(const ASTSelect *root,                                     PlanNode **plan_tree, Status &status) ` |
+| ASTInsertStatement                                           | InsertStmt      | `bool Planner::CreateInsertPlan(const node::ASTInsertStatement*,  *root,                                node::PlanNode **output,                                Status &status) {  // NOLINT (runtime/references)` |
+| ASTCreateStatement                                           | CreateStmt      |                                                              |
+| ASTFunctionDeclaration                                       | FnNodeFnDef     |                                                              |
+
+
+
+## Add new syntax into ZetaSQL (WPI)
+
+ZetaSQL also use bison and flex to parse SQL. It will easy for us to imigrate our New SQL syntax, e.g., `LAST JOIN`, `WINDOW ROWS_RANGE`.
+
+### Last Join
+
+**Solution**:
+
+the `ORDER BY` option can be added into `opt_on_or_using_clause_list`  and add code to deal with it.
+
+### Window ROWS_RANGE
+
+- Make sure HybridSE `udf` and `udaf` works with zetasql - todo
```

Comment:
how to add external udf to zetasql ?

Reply:
Status: todo.
We will to describe it in this section.